apps/ios/SimpleX SE/ShareAPI.swift

```diff
@@ -120,6 +123,12 @@ enum ShareError: LocalizedError {
             "This file format is not supported"
         case .encryptFile:
             "File encryption has failed"
+        case .imageDownsampling:
```

Comment:
let's please get rid of ShareError, and change sendMessage type to:

private func sendMessage() async -> ErrorAlert? {